### PR TITLE
Openamp/Kconfig: add cache and debug config for openamp

### PR DESCRIPTION
**Pull request by CV-Bowen**
## Summary
Commit 1:
openamp/Kconfig: add config to enable/disable the cache feature

Commit 2:
openamp: add OPENAMP_DEBUG config to enable/disable openamp debug

## Impact
None

## Testing
Test with bes board.

### CODE REVIEW

**Review by xiaoxiang781216 — APPROVED** [review]

File: openamp/Kconfig [hunk 1]
```diff
@@ -11,4 +11,19 @@ config OPENAMP
 
 if OPENAMP
 
+config OPENAMP_CACHE
+	bool "Enable OpenAmp cache support"
```
Comment (pkarashchenko) [inline openamp/Kconfig:15]:
```suggestion
	bool "Enable OpenAMP cache support"
```

File: openamp/Kconfig [hunk 2]
```diff
@@ -11,4 +11,19 @@ config OPENAMP
...
+	default n
+	---help---
+		Enable or disable OpenAmp cache support (VIRTIO_CACHED_BUFFERS and
```
Comment (pkarashchenko) [inline openamp/Kconfig:18]:
```suggestion
		Enable or disable OpenAMP cache support (VIRTIO_CACHED_BUFFERS and
```

File: openamp/Kconfig [hunk 3]
```diff
@@ -11,4 +11,19 @@ config OPENAMP
...
+		VIRTIO_CACHED_VRINGS)
+
+config OPENAMP_RPMSG_DEBUG
+	bool "Enable OpenAmp Rpmsg Debug"
```
Comment (pkarashchenko) [inline openamp/Kconfig:22]:
```suggestion
	bool "Enable OpenAMP Rpmsg Debug"
```

File: openamp/Kconfig [hunk 4]
```diff
@@ -11,4 +11,19 @@ config OPENAMP
...
+	default n
+
+config OPENAMP_VQUEUE_DEBUG
+	bool "Enable OpenAmp Virtio Queue Debug"
```
Comment (pkarashchenko) [inline openamp/Kconfig:26]:
```suggestion
	bool "Enable OpenAMP Virtio Queue Debug"
```

File: openamp/Kconfig (hunk 1 above)
Comment (CV-Bowen) [reply openamp/Kconfig]: Done

File: openamp/Kconfig (hunk 2 above)
Comment (CV-Bowen) [reply openamp/Kconfig]: Done

File: openamp/Kconfig (hunk 3 above)
Comment (CV-Bowen) [reply openamp/Kconfig]: Done

File: openamp/Kconfig (hunk 4 above)
Comment (CV-Bowen) [reply openamp/Kconfig]: Done

**Review by pkarashchenko — APPROVED** [review]

**Pull request merged**